dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -315,7 +318,7 @@ private void writeFile(List<HPCCRecord> records, String fileName, FieldDef recor
 
             String clusterName = this.thorcluster;
             System.out.println("Create Start");
-            DFUCreateFileWrapper createResult = dfuClient.createFile(fileName, clusterName, eclRecordDefn, 300, false, DFUFileTypeWrapper.Flat, "");
+            DFUCreateFileWrapper createResult = dfuClient.createFile(fileName, clusterName, eclRecordDefn, connTO, false, DFUFileTypeWrapper.Flat, "");
```

Comment:
Not sure if important, but this test's connection timeout is different now

Reply:
Fixed this so that if connTO is null the original 300 is used.